packages/common-db/src/lib/common/session.ts

```diff
@@ -59,21 +66,22 @@ export function createAdapterStore(session: Session, defaultTtl = 3600): any {
          * @param sess the session
          * @param fn callback
          */
-        set(sid: string, ttl: number, sess: Session, fn: (args: any) => void): void {
+        set(
+            sid: string,
+            ttl: number | SessionData,
+            sess: SessionData | ((err?: Error | null) => void),
+            fn: (err?: Error | null) => void,
+        ): void {
             if (typeof ttl === 'object') {
-                fn = sess;
+                fn = sess as (err?: Error | null) => void;
```

Comment:
if think better check if typef sess === 'function' before hardly defining that type here whoich might be wrong